exampleSite/utilities/README.md

```diff
@@ -59,6 +66,8 @@ Website = ""
 Facebook = ""
 Linkedin = ""
 Pronouns = ""
+Github = ""
+Twitter = ""
```

Comment:
I'm adding that one in the template by default. I don't want them all in the template because I don't want organizers to feel like they have to go collect all of those, but I do want them subtly encouraged to collect the Twitter handles (because it makes livetweeting so much easier).